cfme/tests/cloud/test_provisioning.py

```diff
@@ -103,7 +103,14 @@ def test_provision_from_template_using_rest(
         else provisioning['instance_type'])
     flavours = rest_api.collections.flavors.find_by(name=instance_type)
     assert len(flavours) > 0
-    flavour_id = flavours[0].id
+    # TODO: Multi search when it works
+    for flavour in flavours:
```

Comment:
The rest API uses the 'flavor' spelling, so while you're in here I'd love to see these all get fixed to be consistent with the thing we're testing.